config.json

```diff
@@ -83,33 +83,33 @@
         "difficulty": 1
       },
       {
-        "slug": "raindrops",
-        "name": "Raindrops",
-        "uuid": "7aedb0b6-a4c6-4985-a85c-9b641f4ae047",
+        "slug": "two-fer",
```

Comment:
two-for is usually a good 2nd exercise.

Let's make isogram difficulty 2.

Reply:
`reverse` was also 1, bumping to 2.